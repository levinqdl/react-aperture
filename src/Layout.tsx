import styled from "react-emotion"

const Layout = styled('div')`
  width: 100vw;
  height: 100vh;
`

export default Layout