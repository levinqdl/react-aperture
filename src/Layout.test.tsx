import React from 'react'
import Layout from './Layout'
import { render } from "react-testing-library"

describe('Layout', () => {
  it("renders full viewport", () => {
    const { container, getByText } = render(<Layout><div>Hello Portal Layout!</div></Layout>)
    getByText("Hello Portal Layout!")
    const layout = container.firstChild
    expect(layout).toHaveStyleRule('width', '100vw')
    expect(layout).toHaveStyleRule('height', '100vh')
    expect(layout).toMatchSnapshot()
  })
})