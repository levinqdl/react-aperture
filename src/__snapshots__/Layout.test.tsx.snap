// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders full viewport 1`] = `
.emotion-0 {
  width: 100vw;
  height: 100vh;
}

<div
  class="emotion-0"
>
  <div>
    Hello Portal Layout!
  </div>
</div>
`;
